config/webpack.config.js

```diff
@@ -166,7 +167,8 @@ const coreConfig = function() {
       __dirname,
       '..',
       'webpack/assets/javascripts/all_react_app_exports.js'
-    ),
+    ), 
```

Comment:
Nit: trailing whitespace
```suggestion
    ),
```